CHANGELOG.md

```diff
@@ -7,10 +7,402 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+        -   `listCodesByPreAuthSessionId`
+    -   Added mandatory `tenantId` in the input for the following API interface functions. If any of these functions are overridden, they need to be updated accordingly:
+        -   `createCodePOST`
+        -   `resendCodePOST`
+        -   `consumeCodePOST`
+        -   `emailExistsGET`
+        -   `phoneNumberExistsGET`
+    -   ThirdParty recipe changes
+        -   Removed `signInUp` and added `manuallyCreateOrUpdateUser` instead in the recipe exposed functions.
+        -   Added mandatory `tenantId` as first parameter to the following recipe exposed functions:
+            -   `getUsersByEmail`
+            -   `getUserByThirdPartyInfo`
+        -   Added mandatory `tenantId` in the input for the following recipe interface functions. If any of these functions are overridden, they need to be updated accordingly:
+            -   `getUsersByEmail`
+            -   `getUserByThirdPartyInfo`
+            -   `signInUp`
+        -   Added mandatory `tenantId` in the input for the following API interface functions. If any of these functions are overridden, they need to be updated accordingly:
+            -   `authorisationUrlGET`
+            -   `signInUpPOST`
+        -   Updated `signInUp` recipe interface function in thirdparty with new parameters:
+            -   `oAuthTokens` - contains all the tokens (access_token, id_token, etc.) as returned by the provider
+            -   `rawUserInfoFromProvider` - contains all the user profile info as returned by the provider
+        -   Updated `authorisationUrlGET` API
+            -   Changed: Doesn't accept `clientId` anymore and accepts `clientType` instead to determine the matching config
+            -   Added: optional `pkceCodeVerifier` in the response, to support PKCE
+        -   Updated `signInUpPOST` API
+            -   Removed: `clientId`, `redirectURI`, `authCodeResponse` and `code` from the input
+            -   Instead,
+                -   accepts `clientType` to determine the matching config
+                -   One of redirectURIInfo (for code flow) or oAuthTokens (for token flow) is required
+        -   Updated `appleRedirectHandlerPOST`
+            -   to accept all the form fields instead of just the code
+            -   to use redirect URI encoded in the `state` parameter instead of using the websiteDomain config.
+            -   to use HTTP 303 instead of javascript based redirection.
+    -   Session recipe changes
+        -   Added mandatory `tenantId` as first parameter to the following recipe exposed functions:
+            -   `createNewSession`
+            -   `createNewSessionWithoutRequestResponse`
+            -   `validateClaimsInJWTPayload`
+        -   Added mandatory `tenantId` in the input for the following recipe interface functions. If any of these functions are overridden, they need to be updated accordingly:
+            -   `createNewSession`
+            -   `getGlobalClaimValidators`
```

Comment:
third party recipe change section is not properly indented

Reply:
Done